winlogbeat/_meta/fields.common.yml

```diff
@@ -61,22 +64,26 @@
         The keywords are used to classify an event.
 
     - name: log_name
-      type: keyword
+      # This does not exist yet
+      path: log.name
+      type: alias
```

Comment:
I'd like to understand better the relation between `log_name` and `source_name`. Is there a place where we can see sample data? I haven't found anything obvious.